src/dataframe.rs

```diff
@@ -70,13 +70,14 @@ impl PyDataFrame {
         if let Ok(key) = key.extract::<PyBackedStr>() {
             // df[col]
             self.select_columns(vec![key])
-        } else if let Ok(tuple) = key.extract::<&PyTuple>() {
-            // df[col1, col2, col3]
-            let keys = tuple
-                .iter()
-                .map(|item| item.extract::<PyBackedStr>())
-                .collect::<PyResult<Vec<PyBackedStr>>>()?;
-            self.select_columns(keys)
+        // } else if let Ok(tuple) = key.extract::<&PyTuple>() {
+        // @todo: make this branch work
```

Comment:
This is one place I may need some help on -- it doesn't look like you can extract PyTuples anymore